src/utils/decorators/input-validation/input-validation.js

```diff
@@ -609,7 +608,13 @@ const InputValidation = (ComposedComponent) => {
      */
     get validationHTML() {
       let type = '';
-      if (this.state.valid && !this.state.warning && !this.state.info) { return null; }
+      const validationContext = (
+        <ValidationsContext.Consumer key='validation-context'>
+          { this.initValidationInput }
+        </ValidationsContext.Consumer>
+      );
+
+      if (this.state.valid && !this.state.warning && !this.state.info) { return validationContext; }
```

Comment:
can we remove brackets if it is in one line?
` if (this.state.valid && !this.state.warning && !this.state.info)  return validationContext;`